.github/workflows/test.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v3
         with:
-          go-version: 1.x
+          go-version: '1.19'
```

Comment:
I'm not happy with this.

Reply:
What are you not happy with?

Reply:
Just to have a record here: the `go-test` action was failing, I think in the `check-generate` action, and it seemed to be an issue with the Go version, so I pinned it to what we have in our go.mod. But I didn't like that this was the only PR that was failing that test, and I don't think we want to pin our Go here (since we don't pin it elsewhere).